### ISSUE TITLE
Error - 5_T3_20_create_person_time_monthly.

### ISSUE THREAD

**Issue by anallg** [VAC4EU/CVM#34]
@DavideMessinaARS @mmmartinp @patriciagpoza 

Dear Davide, 

we have a problem in step 5_T3_20. Please, t me know ifyou need more information about it.

_launch_step("p_steps/05_T3_20_create_person_time_monthly.R")
[1] "COUNT PERSON TIME by month"
[1] "PC"
Error in foverlaps(Dummy, Dataset, by.x = c(Increment, "End"), nomatch = 0L,  : 
  All entries in column start_date_of_period should be <= corresponding entries in column end_date_of_period in data.table 'y'.
In addition: Warning message:
In CountPersonTime(Dataset = study_population, Person_id = "person_id",  :
 
 Error in foverlaps(Dummy, Dataset, by.x = c(Increment, "End"), nomatch = 0L, : 
All entries in column start_date_of_period should be <= corresponding entries in column end_date_of_period in data.table 'y'.
10.
stop("All entries in column ", yintervals[1L], " should be <= corresponding entries in column ", 
yintervals[2L], " in data.table 'y'.")
9.
foverlaps(Dummy, Dataset, by.x = c(Increment, "End"), nomatch = 0L, 
type = "any") at CountPersonTimeV13.6.R#167
8.
CountPersonTime(Dataset = study_population, Person_id = "person_id", 
Start_study_time = gsub("-", "", as.character(study_start)), 
End_study_time = gsub("-", "", as.character(max_exit)), Start_date = "start_date_of_period", 
End_date = "end_date_of_period", Birth_date = "date_of_birth", ... at 05_T3_20_create_person_time_monthly.R#31
7.
eval(ei, envir)
6.
eval(ei, envir)
5.
withVisible(eval(ei, envir))
4.
source(paste0(thisdir, "/", x))
3.
system.time(source(paste0(thisdir, "/", x)))
2.
print(system.time(source(paste0(thisdir, "/", x)))) at launch_step.R#9
1.
launch_step("p_steps/05_T3_20_create_person_time_monthly.R")
Timing stopped at: 224.3 59.57 227.2_

**Comment by DavideMessinaARS** [comment]
Run the following command in the console after loading the dataset D3_study_population_by_dose_PC in g_intermediate:

```r
test <- D3_study_population_by_dose_PC[start_date_of_period > end_date_of_period, ]
```

Then check if the dataset is empty (empty = good). If not, I'm afraid we will need a call to understand why are those persons there. (maybe children, so exclusion criteria)

**Comment by elisacm98** [comment]
Hi,

We have been running the code and we have the same error. In our case is with the subpopulation "ALL". I have also run the following command : 

test <- D3_study_population_by_dose[start_date_of_period > end_date_of_period, ]

The dataset is not empty, we have one person.

**Comment by DavideMessinaARS** [comment]
I've fixed the issue in 918d8af6f41d669224b3b83da674559fe6c9ce63.
I'm preparing a new release
